### PR TITLE
wyebadblock: new package at git-version

### PR DESCRIPTION
**Pull request by sdsddsd1**
## Description of package
Wyebadblock is a adblock extension for webkit2gtk based browsers.
See [github page](https://github.com/jun7/wyebadblock) for documentation.

## Installed manifest of package

```
/var/db/kiss/installed/wyebadblock/version
/var/db/kiss/installed/wyebadblock/sources
/var/db/kiss/installed/wyebadblock/post-install
/var/db/kiss/installed/wyebadblock/manifest
/var/db/kiss/installed/wyebadblock/depends
/var/db/kiss/installed/wyebadblock/checksums
/var/db/kiss/installed/wyebadblock/build
/var/db/kiss/installed/wyebadblock/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/wyebrowser/adblock.so
/usr/lib/wyebrowser/
/usr/lib/
/usr/bin/wyebab
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/wyebadblock/build [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+make
+
```
Comment (dylanaraps) [inline community/wyebadblock/build:4]: Remove this newline.

File: community/wyebadblock/post-install [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cat <<EOF
+
+Installation instructions:
+
+Wyebadblock is looking for adblocking rules at $Home/.config/wyebadblock/easylist.txt
```
Comment (dylanaraps) [inline community/wyebadblock/post-install:7]: Should be `$HOME`

Comment (dylanaraps) [reply community/wyebadblock/post-install:7]:
```
Wyebadblock looks for adblocking rules in
```

File: community/wyebadblock/post-install [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Documentation can be found at https://github.com/jun7/wyebadblock
+
+surf:
+		mkdir usr/lib/surf
```
Comment (dylanaraps) [inline community/wyebadblock/post-install:11]: `/usr/lib/surf`

**Comment by sdsddsd1** [comment]
updated

File: community/wyebadblock/depends [hunk 4]
```diff
@@ -0,0 +1 @@
+webkit2gkt
```
Comment (dylanaraps) [inline community/wyebadblock/depends]: Typo. Should be `webkit2gtk` (You have `2gkt`).

Comment (sdsddsd1) [reply community/wyebadblock/depends]: I have allready fixed that  and pushed a pr. Not there?

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**